src/ProjectManager.cpp

```diff
@@ -270,9 +269,9 @@ void InitProjectWindow( ProjectWindow &window )
    //      will be given the focus even if we try to SetFocus().  By
    //      making the TrackPanel that first window, we resolve several
    //      keyboard focus problems.
-   pPage->MoveBeforeInTabOrder(topPanel);
+   window.GetContainerWindow()->MoveAfterInTabOrder(topPanel);
 
-   bs = (wxBoxSizer *)pPage->GetSizer();
+   bs = (wxBoxSizer *)window.GetTrackListWindow()->GetSizer();
```

Comment:
Clean this up as a modern `static_cast` while you are visiting the neighborhood.

Reply:
Ok